internal/identify/performer.go

```diff
@@ -24,6 +27,12 @@ func getPerformerID(ctx context.Context, endpoint string, w PerformerCreator, p
 
 		return &performerID, nil
 	} else if createMissing && p.Name != nil { // name is mandatory
+		// skip single name performers with no disambiguation
+		if skipSingleNamePerformers && !strings.Contains(*p.Name, " ") && (p.Disambiguation == nil || len(*p.Disambiguation) == 0) {
+			// hack to let the upstream code know a performer was skipped
+			id := int(-1)
```

Comment:
I think returning and checking for an `ErrSkipSingleNamePerformer` error might be the better approach here.